src/illuminatio/illuminatio_runner.py

```diff
@@ -167,25 +167,14 @@ def run_tests_for_target(network_ns, ports, target):
     LOGGER.info("Target: %s", target)
     port_on_nums = {port.replace("-", ""): port for port in ports}
     port_string = ",".join(port_on_nums.keys())
-    # ToDo do we really need this -> we know the service already and could use the cluster IP
-    # DNS could be blocked
-    # Only IPv4 currently -> https://docs.python.org/3/library/socket.html#socket.getaddrinfo
-    svc_dns_entry = get_domain_name_for(target)
-    LOGGER.info(svc_dns_entry)
-    try:
-        svc_ip = socket.gethostbyname(svc_dns_entry)
-    except (socket.gaierror, socket.herror) as error:
```

Comment:
You can remove the socket import now: https://travis-ci.org/github/inovex/illuminatio/jobs/719666175#L331

Reply:
Thanks for pointing that out, removed it